### ISSUE TITLE
error: cannot convert ‘std::basic_istream<char>::__istream_type’ to bool

### ISSUE THREAD

**Issue by smoe** [alexdobin/STAR#957]
I thought I better don't mess with this one:
```
g++ -c -Wdate-time -D_FORTIFY_SOURCE=2  -O3 -std=c++11 -fopenmp -D'COMPILATION_TIME_PLACE="<not set in Debian>"' -flto -flto -g -O2 -fdebug-prefix-map=/home/moeller/git/med-team/rna-star=. -fstack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=2 -std=c++0x -g -O2 -fdebug-prefix-map=/home/moeller/git/med-team/rna-star=. -fstack-protector-strong -Wformat -Werror=format-security -D'COMPILATION_TIME_PLACE="<not set in Debian>"' SuperTranscriptome.cpp
SuperTranscriptome.cpp: In member function ‘void SuperTranscriptome::load(char*, std::vector<long long unsigned int>&, std::vector<long long unsigned int>&)’:
SuperTranscriptome.cpp:51:34: error: cannot convert ‘std::basic_istream<char>::__istream_type’ {aka ‘std::basic_istream<char>’} to ‘bool’ in initialization
   51 |         bool inGood = (superTrSJ >> sutr);
      |                       ~~~~~~~~~~~^~~~~~~~
      |                                  |
      |                                  std::basic_istream<char>::__istream_type {aka std::basic_istream<char>}

```
Is a ">0" missing?

Spotted by
```
$ g++ --version
g++ (Debian 9.3.0-14) 9.3.0
Copyright (C) 2019 Free Software Foundation, Inc.
```

**Comment by alexdobin** [comment]
Hi Steffen,

I think this has been fixed: #948, please try the latest patch from GitHub master.

Cheers
Alex

**Comment by smoe** [comment]
Worked! Many thanks!
